javascript/ql/src/semmle/javascript/PackageExports.qll

```diff
@@ -18,7 +28,7 @@ private int countSlashes(string path) { result = count(path.splitAt("/")) - 1 }
  * There can be multiple results if the there exists multiple package.json that are equally deeply nested in the folder structure.
  * Results are limited to package.json files that are at most nested 2 directories deep.
  */
-PackageJSON getTopmostPackageJSON() {
+private PackageJSON getTopmostPackageJSON() {
```

Comment:
This visibility change is a breaking change.

Reply:
I made it public again.  
(Making it private also broke a test).